sticker.html

```diff
@@ -59,6 +59,7 @@
 <div class="container main-content">
   <div class="splash text-center">
     <p>You can order stickers from the following places:</p>
+    <p><a class="bold" href="http://www.hexpace.net/">hexpace</a> a pure tech-based hexagon sticker market who sells premium durable tech stickers . They ship internationally.</p>
```

Comment:
Thanks for the PR. Could you move this below the links that do custom printing? I'd like to keep those at the top, since many people will be looking to print their own stickers.

Reply:
Please review the latest commit